appium-tests/common/common.spec.js

```diff
@@ -242,7 +248,6 @@ describe('Contacts Android', function () {
 
     describe('Picking contacts', function () {
         afterEach(function (done) {
-            checkSession(done);
```

Comment:
>we could probably get away with checking the session only immediately after the initial should connect to an appium endpoint properly spec
And how would all other specs then know that they should skip any Appium interactions?